packages/react-charting/src/utilities/testData.ts

```diff
@@ -0,0 +1,119 @@
+import { DefaultPalette } from '@fluentui/react/lib/Styling';
+import { IVSChartDataPoint, IVerticalStackedChartProps } from '../index';
```

Comment:
rename to test-data.test.ts and include in npmignore

Reply:
Are we not maintaining camel casing in case of file names?

Reply:
using kebab casing is the norm for react typescript and followed across fluent. Not sure why we are following camel casing. We can start with kebab casing for the tests at least.